### ISSUE TITLE
ONIX Screen Reader Friendly

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-a11y#17]
Can be identified by following the [crosswalk](http://www.a11ymetadata.org/the-specification/metadata-crosswalk/):

```
accessModeSufficient: textual
```

equals

```
List: 81; Code: 10 combined with List: 196; Code: 10 means all text is actual text. Note that List: 81; Code: 10 on its own (without List: 196; Code: 10) admits the possibility that the ‘text’ is inaccessible because it is an image of text.
```

**Comment by editadapt** [comment]
Reviewing the crosswalk we found this mapping insufficient.

Our understanding is as follow : 
* 81 code 10 _Text (eye-readable) ... designate  an e-book or other digital or physical product whose primary content is  eye-readable text_ does not represents a thru text content. Exemple : a picture book with full descriptions will not get 81/10 marked because it's not _Text (eye-readable)_ (primary content is graphical) but will still be Accessible because it has true text alternatives.
* 196 code 10 adress Reading system _(No reading system accessibility options disabled)_ and refers mainly to DRM action.
* Alternatives to graphical content is not represented, those are adressed by 196 codes 14, 15 and 16.

We found no easy way to represent this information in ONIX. 
* The best representation we found is a reasoning by absence of 196 code 09 _(Known to lack significant features required for broad accessibility.)_ Any publication without 196/09 would be Screen Reader Friendly. 
*  Another option could be 196 with one or more of 14, 15, 16.

Both ways requires a strong capacity building in publishing teams.

Our proposal is to write and send a request to Editeur for the addition of a "True text content" code in list 81 or 196.

**Comment by gautierchomel** [comment]
adressed in recent ONIX issues
